test/resource_unittest.cc

```diff
@@ -0,0 +1,55 @@
+#include "../src/resource.h"
+#include "../src/json.h"
+#include "gtest/gtest.h"
+
+namespace {
+
+TEST(ResourceTest, Type) {
+  google::MonitoredResource mr("some_resource", {});
+  EXPECT_EQ("some_resource", mr.type());
+}
+
+TEST(ResourceTest, Labels) {
+  google::MonitoredResource mr("", {{"foo", "bar"}, {"bar", "baz"}});
+  EXPECT_EQ(2, mr.labels().size());
+  EXPECT_EQ("bar", mr.labels().at("foo"));
+  EXPECT_EQ("baz", mr.labels().at("bar"));
+}
+
+TEST(ResourceTest, BasicEquality) {
+  google::MonitoredResource mr1("", {});
+  google::MonitoredResource mr2("", {});
+
+  EXPECT_EQ(mr1, mr2);
+}
+
+TEST(ResourceTest, BasicTypeComparison) {
+  google::MonitoredResource mr1("2", {});
+  google::MonitoredResource mr2("1", {});
+
+  EXPECT_LT("2", "1");
```

Comment:
Leftover debug code?

Reply:
Removed. It was as an example (for my mistaken understanding of the comparison)